### PR TITLE
ENH: removed pyqi

### PR DESCRIPTION
**Pull request by ebolyen**
:tada: :cake: :confetti_ball:

### CODE REVIEW

**Comment by ebolyen** [comment]
build is upset that `biom` script is gone. Shall we wait until `biomcli` is complete to merge this? Then we can rename it here.

**Comment by gregcaporaso** [comment]
yep, we need to hold off. i'm working on convert now

File: ChangeLog.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ BIOM-Format ChangeLog
 biom 2.1.4-dev
 --------------
 
+Changes:
+
+* Codebase now uses [click](http://click.pocoo.org) instead of
```
Comment (wasade) [inline ChangeLog.md]: :cocktail:

**Comment by wasade** [comment]
-4200 lines? wow. And PR 666...

**Comment by gregcaporaso** [comment]
This looks good, thanks @ebolyen! Merging... :beers:

**Pull request merged**